app/views/maestros/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+
+<%= form_for @maestro do |f| %>
```

Comment:
:+1: for using a partial for the form; however, it is best practice to pass `@maestro` down as a local variable instead of using the instance variable, itself.
```erb
<%= render 'form', maestro: @maestro %>
```
for example